string checkerName
time stamp
---
bool response
uint64[] buffer_p
uint64[] buffer_o
uint32[] timerBounds
float64[] timerInstances
bool[] timerCache
uint64[] currAss
uint64[] nextAss
uint64[] currAnt
uint64[] nextAnt
uint32 conIns
uint32 endIns
bool last_p0
bool last_p1
bool last_p2
time last_msg_ts

